packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -322,6 +333,88 @@ class _MethodStreamManager {
     );
   }
 
+  void _handleMethodStreamEndpoint(
+    MethodStreamConnector methodConnector,
+    Session session,
+    Map<String, dynamic> args,
+    OpenMethodStreamCommand message,
+    Server server,
+  ) {
+    methodConnector.call(session, args).listen(
+      (value) {
+        _postMessage(
+          endpoint: message.endpoint,
+          method: message.method,
+          connectionId: message.connectionId,
+          message: MethodStreamMessage.buildMessage(
+            endpoint: message.endpoint,
+            method: message.method,
+            connectionId: message.connectionId,
+            object: server.serializationManager.encodeWithType(value),
```

Comment:
Suggestion: What do you think about `data` rather than `object`.

Reply:
No specific opinio.

In this case we are actually serializing an object since it has Class name and all possible fields. So we don't take any general data.

Reply:
Wont fix for now but open for a better name later on.